{
  "id": "CVE-2023-1103",
  "sourceIdentifier": "security@huntr.dev",
  "published": "2023-03-01T02:15:51.417",
  "lastModified": "2023-04-04T20:15:07.343",
  "vulnStatus": "Rejected",
  "descriptions": [
    {
      "lang": "en",
      "value": "** REJECT ** DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Reason: This candidate was withdrawn by its CNA. Further investigation showed that it was a duplicate of CVE-2022-4821. Notes: none."
    }
  ],
  "metrics": {},
  "references": []
}